### ISSUE TITLE
Make a BattleScribe 1.15 version available

### ISSUE THREAD

**Issue by Obiken** [BSData/warmahordes#48]
Would be nice to provide a *.bsr file for people to play with, see example at WH 40K.

**Comment by amis92** [comment]
Looking about, isn't latest release still 1.15? Or did you open the issue in case (per #47) the repo gets some maintenance and upgrades to 2.0?

**Comment by Obiken** [comment]
I opened the issue before I looked into the delivery method. But I see other repos just tag the right version and move on from there. So no need for the issue really if no one feels a need to do some clean up?

**Comment by amis92** [comment]
Not exactly. In 40k and Horus Heresy (if I remember correctly) we've tagged last release for 1.15, but also we've added an attachment to that release. That attachment is a .bsr repo packaged using BSv1.15 Data Indexer. That's the key difference and the additional work to be done. It allows users to download it and directly import into BattleScribe, instead of manually unzipping auto-attachments.
